tests/integration/test_fim/test_files/test_audit/test_audit_multiple_keys.py

```diff
@@ -0,0 +1,145 @@
+
+test_directories = [monitored_test_dir, non_monitored_test_dir]
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+configurations_path = os.path.join(test_data_path, 'wazuh_conf_multiple_audit_keys.yaml')
+
+wazuh_log_monitor = FileMonitor(fim.LOG_FILE_PATH)
+
+# Configurations
+
+config_params, config_metadata = fim.generate_params(extra_params={'MONITORED_DIR': monitored_test_dir,
+                                                     'AUDIT_KEYS': ','.join(key for key in custom_keys)},
+                                                     modes=['whodata'])
+
+configurations = load_wazuh_configurations(configurations_path, __name__,
+                                           params=config_params, metadata=config_metadata)
+
+
+def run_audit_command(directory, params='', cmd_type='add'):
+    """Function to run auditctl commands.
+
+    Args:
+        directory (str): An audit rule will be added or deleted for this directory.
+        params (str): Parameters for auditctl.
+        cmd_type (str): type of command that will be used.
+                        - add to create a new audit rule.
+                        - delete to remove an audit rule.
+    Raises:
+        ValueError: If cmd_type is not add or delete.
+    """
+
+    if cmd_type == 'add':
+        audit_flag = '-w'
+    elif cmd_type == 'delete':
+        audit_flag = '-W'
+    else:
+        raise ValueError("Only add and delete values are allowed")
+
+    command = f"auditctl {audit_flag} {directory} -p wa {params}"
+    subprocess.run(command.split())
+
+
+# fixtures
+
+@pytest.fixture(scope='module', params=configurations)
+def get_configuration(request):
+    """Get configurations from the module."""
+    return request.param
+
+
+# Functions
+
+def extra_configuration_before_yield():
+    # Create the custom audit rules for the non monitored directory
+    run_audit_command(directory=non_monitored_test_dir, params=param_list, cmd_type='add')
+    # Remove audit rule that FIM configures for each monitored directory
+    run_audit_command(directory=monitored_test_dir, params='-k wazuh_fim', cmd_type='delete')
+    # Set the audit rule for the monitored directory with more than one key
+    run_audit_command(directory=monitored_test_dir, params='-k wazuh_fim -k "a_random_key"', cmd_type='add')
+
+
+def extra_configuration_after_yield():
+    # Remove the audit rules configured by the test
+    run_audit_command(directory=non_monitored_test_dir, params=param_list, cmd_type='delete')
+    run_audit_command(directory=monitored_test_dir, params=param_list, cmd_type='delete')
+
+
+@pytest.mark.parametrize('tags_to_apply', [{'audit_multiple_keys'}])
+@pytest.mark.parametrize('directory', [monitored_test_dir, non_monitored_test_dir])
+def test_audit_multiple_keys(directory, tags_to_apply, get_configuration, configure_environment, restart_syscheckd,
+                             wait_for_fim_start):
```

Comment:
After talking with @snaow we have considered that new development or changes into existing tests should follow new repository standards about Wazuh daemons. For now, it is required to use fixtures to start only required daemons, and turn them off when the test is over.  That means, that the default status of Wazuh would be off and each test should be responsible for daemons handling.
In case of the need of all Wazuh daemon, the test should disable every module not necessary for the testing in `ossec.conf`.

The reasons for this change are:
- Faster testing
- More encapsulated tests